pysatNASA/instruments/de2_vefi.py

```diff
@@ -89,10 +114,61 @@
 list_files = functools.partial(mm_gen.list_files,
                                supported_tags=supported_tags)
 
+
 # Set the load routine
-# Forcing use of cdflib as default since pysatCDF has a known issue with vefi
-# data.  See pysat/pysatCDF#48
-load = functools.partial(cdw.load, use_cdflib=True)
+def load(fnames, tag='', inst_id='', **kwargs):
+    """Load DE2 VEFI data.
+
+    This routine is called as needed by pysat. It is not intended
+    for direct user interaction.
+
+    Parameters
+    ----------
+    fnames : array-like
+        Iterable of filename strings, full path, to data files to be loaded.
+        This input is nominally provided by pysat itself.
+    tag : str
+        Tag name used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself. (default='')
+    inst_id : str
+        Instrument ID used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself. (default='')
+
+    Returns
+    -------
+    data : pds.DataFrame
+        A pandas DataFrame with data prepared for the `pysat.Instrument`.
```

Comment:
Update to note support for both pandas and xarray as well as what conditions each format is returned.

Reply:
This only returns pandas as output. xarray is used to convert the data for the blank tag, which is deprecated.